examples/lib/defaultHtmlStepUi.js

```diff
@@ -156,6 +157,7 @@ function DefaultHtmlStepUi(_sequencer, options) {
       $(step.ui.querySelectorAll(".cal")).toggleClass("collapse");
     });
     
+    $(step.imgElement).on("mousemove", debounce(() => imageHover(step), 150));
```

Comment:
What does debounce do?

Reply:
it limits a fn being called too frequently , so that site performance isnt hampered

Reply:
Cool